Sources/PointFree/Database.swift

```diff
@@ -743,6 +796,30 @@ private func migrate() -> EitherIO<Error, Prelude.Unit> {
       "created_at" timestamp without time zone DEFAULT NOW() NOT NULL
       """
     )))
+    .flatMap(const(execute(
+      """
+      ALTER TABLE "users"
+      ADD COLUMN IF NOT EXISTS
+      "rss_salt" uuid DEFAULT uuid_generate_v1mc() NOT NULL
```

Comment:
the salt used to validate urls for a user's private RSS feed. we can rotate this salt to invalidate any urls out in the world.